modules/andes-core/broker/src/main/java/org/wso2/andes/store/FailureObservingMessageStore.java

```diff
@@ -531,6 +531,14 @@ public void removeQueue(String storageQueueName) throws AndesException {
      * {@inheritDoc}
      */
     @Override
+    public void removeCachedQueue(String storageQueueName) {
```

Comment:
Conceptually caching is internal to message store isn't? 
Is there a possibility handle this with in the message store impl?